image/image.go

```diff
@@ -115,6 +122,10 @@ func Unpack(tarFile, dest, ref string) error {
 }
 
 func unpack(w walker, dest, refName string) error {
+	if err := refsNameRegexpCheck(refName); err != nil {
```

Comment:
OK, I understand your concern. I will change it after [PR](https://github.com/opencontainers/image-spec/pull/421) in image-spec is merged, which leads to CI failure without merging it.